src/Tools/Replay/Replay.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>$(NetRoslyn);net472</TargetFrameworks>
+    <GenerateMicrosoftCodeAnalysisCommitHashAttribute>true</GenerateMicrosoftCodeAnalysisCommitHashAttribute>
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\Compilers\Core\CommandLine\BuildProtocol.cs" />
+    <Compile Include="..\..\Compilers\Core\CommandLine\CompilerServerLogger.cs" />
+    <Compile Include="..\..\Compilers\Core\CommandLine\NativeMethods.cs" />
+    <Compile Include="..\..\Compilers\Core\Portable\CommitHashAttribute.cs" />
+    <Compile Include="..\..\Compilers\Core\Portable\InternalUtilities\PlatformInformation.cs" />
+    <Compile Include="..\..\Compilers\Core\Portable\InternalUtilities\Debug.cs" />
+    <Compile Include="..\..\Compilers\Core\Portable\InternalUtilities\NullableAttributes.cs" />
+    <Compile Include="..\..\Compilers\Shared\BuildServerConnection.cs" />
+    <Compile Include="..\..\Compilers\Shared\NamedPipeUtil.cs" />
+    <Compile Include="..\..\Compilers\Shared\RuntimeHostInfo.cs" />
```

Comment:
It bothers me a bit that all of these files are needed to effectively run the client side of the build server. Considering changing this in a future PR to be a lot simpler. Essentially: 

1. Create a shared project that has all of this logic in it.
2. Reference that shared project in MS.CA. Then vbc, csc and VBCSCompiler can just use the logic as they have IVT to MS.CA.
3. Reference that shared project in MS.Build.Tasks.CA.csproj. It does not reference MS.CA (by very explicit design) hence needs its own copy of the source.
4. Reference that project in MS.CA as well.